src/app/dim-ui/Switch.m.scss

```diff
@@ -59,12 +59,13 @@ $touchPadding: 2px; // A bit of extra hit area
     &:checked {
       & + label {
         &::before {
-          background-color: change-color($orange, $alpha: 0.5);
+          background-color: var(--theme-accent-primary);
+          opacity: 0.5;
```

Comment:
I believe we want only the background to be transparent, not the whole element.

Reply:
This was another instance where i could not pass a css var into a mixin/function to only adjust the background tint, so this was the fallback option. 

The component looks/works the same — I don't believe there's any adverse consequences of taking this approach. 

There is only 1 other occurrence of the orange being manipulated to a different tint, neither of these are worth maintaining a variable for IMO.

Reply:
Maybe someday: https://caniuse.com/css-relative-colors

Otherwise yeah as long as we're careful about it.